src/Constants/actions.js

```diff
@@ -0,0 +1,16 @@
+const ACTIONS = {
+  DISPLAY_PRODUCTS: "display-products",
+  HIGH_TO_LOW: "high-to-low",
+  LOW_TO_HIGH: "low-to-high",
+  ALL_GENDERS: "all-genders",
+  MALE_GENDER: "male-gender",
+  FEMALE_GENDER: "female-gender",
+  FILTER_BY_SIZE: "filter-by-size",
+  ONLY_INSTOCK: "only-in-stock",
+  ONLY_FAST_DELIVERY: "only-fast-delivery",
+  PRICE_RANGE_SLIDER: "price-range-slider",
+  CLEAR_FILTER: "clear-filter",
+  FILTER_BY_BRAND: "filter-by-brand",
+};
```

Comment:
Appreciation: Great you have created the variables for reducer type to avoid spelling mistakes. Great work.